test/integration/install_test.go

```diff
@@ -349,7 +351,7 @@ func TestInstallOrUpgradeCli(t *testing.T) {
 		}
 
 		// prepare for stage 2
-		args = append([]string{"control-plane", "--addon-overwrite"}, args...)
+		args = append([]string{"control-plane", "--skip-outbound-ports", skippedOutboundPorts}, args...)
```

Comment:
Also can you add here helm-style flags as well?

Reply:
I don't think so. because this part is invoked both with `upgrade-edge` and `upgrade-stable` which would mean `upgrade-stable would fail`. Is there a way to gate this to be only for the edges?

Reply:
.... well, there's `TestHelper.UpgradeFromVersion()`, which could be augmented through some wrapper checking if it corresponds to an edge or a stable...

Reply:
Ahh, Let me try. Also, Def feels like an important test to check if things are actually persisted after upgrades.